hack/util.sh

```diff
@@ -609,9 +617,9 @@ function install_router {
                  --cert="${MASTER_CONFIG_DIR}/router.crt" --key="${MASTER_CONFIG_DIR}/router.key"
             cat "${MASTER_CONFIG_DIR}/router.crt" "${MASTER_CONFIG_DIR}/router.key" \
                 "${MASTER_CONFIG_DIR}/ca.crt" > "${MASTER_CONFIG_DIR}/router.pem"
-            ROUTER_DEFAULT_CERT="--default-cert=${MASTER_CONFIG_DIR}/router.pem"
+            router_args+=("--default-cert='${MASTER_CONFIG_DIR}/router.pem'")
```

Comment:
I was intending this - don't we want the quotation marks so that the flag value gets parsed as one item in the case that `$MASTER_CONFIG_DIR` has whitespace in it?